src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -280,6 +280,9 @@ export default {
 		// Update default value
 		this.updateContent(this.value)
 
+		// Removes the contenteditable attribute at first load if the prop is
+		// set to false.
+		this.$refs.contenteditable.contentEditable = this.contenteditable
```

Comment:
```suggestion
		this.$refs.contenteditable.contenteditable = this.contenteditable
```

Reply:
Nope, that's the name of the actual key of a div.contenteditable html element. It's a weird convention given that in the template it's all lowercase 🤷

![Screenshot from 2020-11-20 10-50-40](https://user-images.githubusercontent.com/26852655/99786108-60eba000-2b1e-11eb-9469-fb96eb8ab9fa.png)

Reply:
That is awful ^^'
https://developer.mozilla.org/fr/docs/Web/API/HTMLElement/contentEditable